backend/awsvpc/awsvpc.go

```diff
@@ -162,6 +166,15 @@ func (m *AwsVpcBackend) Init(extIface *net.Interface, extIP net.IP) (*backend.Su
 	}, nil
 }
 
+func (m *AwsVpcBackend) DisableSrcDestCheck(instanceID string, ec2c *ec2.EC2) (*ec2.ModifyInstanceResp, error) {
+	modifyAttributes := &ec2.ModifyInstance{
+		SourceDestCheck:    false,
+		SetSourceDestCheck: true,
+	}
+
+	return ec2c.ModifyInstance(instanceID, modifyAttributes)
```

Comment:
I'm assuming this won't fail if the source/dest check is already disabled, correct?